src/components/AddOrEditSkillModale.tsx

```diff
@@ -83,7 +93,9 @@ const AddOrEditSkillModale = ({
             {[1, 2, 3, 4, 5].map((index) => (
               <button
                 key={`desire-${index}`}
-                onClick={() => setDesireLevel(index)}
+                onClick={() => {
+                  setDesireLevel(index);
+                }}
```

Comment:
This is causing a merge conflict for some reason, I think we should revert this change since it doesn't bring anything.
Then we can merge 😃

Reply:
done 👍